buildscripts/docker/Dockerfile.windows_builder

```diff
@@ -0,0 +1,236 @@
+#    Copyright © 2021 by The qTox Project Contributors
+#
+#    This program is libre software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+FROM debian:bullseye-slim
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+ARG ARCH
+ARG WINEARCH
+ENV WINEARCH=${WINEARCH}
+
+RUN dpkg --add-architecture i386 && \
+    apt-get update && apt-get install -y --no-install-recommends \
+                   autoconf \
+                   automake \
+                   build-essential \
+                   ca-certificates \
+                   cmake \
+                   extra-cmake-modules \
+                   git \
+                   libarchive-tools \
+                   libtool \
+                   nsis \
+                   pkg-config \
+                   python3-pefile \
+                   tclsh \
+                   texinfo \
+                   unzip \
+                   curl \
+                   yasm \
+                   zip \
+                   g++-mingw-w64-${ARCH//_/-} \
+                   gcc-mingw-w64-${ARCH//_/-} \
+                   gdb-mingw-w64 \
```

Comment:
Why was `gdb-mingw-w64` package added?

Reply:
This is really useful for debugging CI failures. Pop open the docker image and run a failing test with winedbg/gdb. Yeah it's a little extra overhead, but I think it's worth it for ease of debugging

Reply:
Ok, makes sense.